Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -1126,8 +1172,19 @@ function vtkRenderWindowInteractor(publicAPI, model) {
     model.currentRenderer = r;
   };
 
+  publicAPI.setContainer = (container) => {
+    if (container === undefined) {
+      vtkErrorMacro('Container cannot be undefined.');
+    }
```

Comment:
Did you mean to return here or just show a warning message? Another option is to slightly relax this condition by removing this if statement and running `setContainer(container ?? null)` below to always get null.